### PR TITLE
Wnmgds 1832 build system cleanup

### PR DESCRIPTION
**Pull request by forrestbaer**
## Summary

- Removes the `design-system-scripts` package from the repo
- Moves all testing output from `packages` to appropriate folders in `/tests`
- Removes `stats` scripts
- Moves all `devDependencies` into monorepo root
- Adds single `gulpfile.js` for build system
- Updated documentation to reflect changes

## How to test

1. `rm -rf node_modules && yarn install && yarn build && yarn build:healthcare && yarn build:medicare`
2. Inspect output in each packages `/dist` folder
3. `yarn test && yarn type-check`

## Checklist

- [x] Prefixed the PR title with the [Jira ticket number](https://jira.cms.gov/projects/WNMGDS/) as `[WNMGDS-####] Title`

### If this is a change to code:

- [x] Created or updated unit tests to cover the logic added
- [x] If necessary, updated unit-test snapshots (`yarn test:unit:update`) and browser-test snapshots (`yarn test:browser:update`)

### If this is a change to documentation:

- [x] Checked for spelling and grammatical errors

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,15 +16,14 @@
...
-| [CMSDS Documentation](packages/docs)                       | Gatsby based CMSDS Documentation site.                                                                                                                                                                                                                                                                                                                                                                                                                       |
+You're currently at the root of a monorepo containing multiple NPM packages located in the [`packages` directory](packages/). Unless you're a contributor or a child design system maintainer. View the `README.md` in each of these for additional details.
```
Comment (pwolfert) [inline README.md:19]: Looks like this whole sentence can be removed

```suggestion
You're currently at the root of a monorepo containing multiple NPM packages located in the [`packages` directory](packages/). View the `README.md` in each of these for additional details.
```

File: packages/ds-healthcare-gov/package.json [hunk 2]
```diff
@@ -28,12 +28,6 @@
     "js-cookie": "^2.2.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.0.0",
-    "@babel/plugin-proposal-class-properties": "^7.10.1",
-    "@babel/plugin-transform-object-assign": "^7.10.4",
-    "@babel/preset-typescript": "^7.15.0",
-    "@cmsgov/design-system-scripts": "4.0.0",
-    "babel-eslint": "^10.1.0",
```
Comment (pwolfert) [inline packages/ds-healthcare-gov/package.json:36]: Nice 🗑️

File: package.json [hunk 3]
```diff
@@ -27,7 +27,7 @@
     "storybook": "start-storybook -p 6006",
     "test": "yarn test:unit && yarn test:a11y",
     "posttest": "yarn lint",
-    "test-coverage": "yarn cmsds test ./packages --collectCoverage",
+    "test:coverage": "yarn test:unit --collectCoverage",
```
Comment (pwolfert) [inline package.json:30]: Nice 👍. Maybe it can be even more specific, like `test:unit:coverage`

**Review by pwolfert — COMMENTED** [review]
This is amazing! I haven't gotten to the actual build script yet. Just reviewing files on the periphery first

**Review by pwolfert — APPROVED** [review]
New build script looks great

File: package.json (hunk 3 above)
Comment (forrestbaer) [reply package.json]: Good call

**Pull request merged**